test/lib/index.test.ts

```diff
@@ -324,6 +324,12 @@ describe('Testing behaviors with issue(s)', () => {
 \t+ Via:   goof@0.0.3 => express-fileupload@0.0.5 => @snyk/nodejs-runtime-agent@1.14.0 => acorn@5.7.3
 \t+ Fixed in: acorn, 5.7.4, 6.4.1, 7.1.1
 \t+ Fixable by upgrade: false=>@snyk/nodejs-runtime-agent@1.14.0=>acorn@5.7.4
+## License
+1 issue found 
```

Comment:
what does it look like when 2 licenses are found?  Not familiar with this repo and  https://github.com/snyk-tech-services/snyk-prevent-gh-commit-status/pull/22/files#diff-07e6891656defce09b8e5fe5d0f3cea3fb570f75368d71793de0097900ef33e4R94 looks to me like it will do 
```
1/1: 
      Artistic-2.0 license 
      [Medium Severity]
1/2
      Apache-2.0 license 
      [Critical Severity]
```
is that what you are going for here?

Reply:
didn't notice they is one missing... Thanks